meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb

```diff
@@ -2,7 +2,7 @@ DESCRIPTION = "Mender Application Update Module for supporting containerized app
 
 HOMEPAGE = "https://mender.io"
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=b4b4cfdaea6d61aa5793b92efd42e081"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=30b4554c64108561c0cb1c57e8a044f0"
```

Comment:
This one also should not be updated without updating `SRCREV`. The two go hand in hand.

Reply:
Thanks. I got it now.

If I drop these two I can actually run the pipeline too